# The AUVSI map

AuvsiBoundaries[] bds
AuvsiStaticObstacle[] obs
rosplane_msgs/Waypoint[] wps
rosplane_msgs/Waypoint[] primary_wps
